packages/flutter/lib/src/painting/box_painter.dart

```diff
@@ -1051,7 +1051,7 @@ class BoxDecoration extends Decoration {
   /// * If [backgroundColor] is null, this decoration does not paint a background color.
   /// * If [backgroundImage] is null, this decoration does not paint a background image.
   /// * If [border] is null, this decoration does not paint a border.
-  /// * If [borderRadius] is null, this decoration use more efficient background
+  /// * If [borderRadius] is null, this decoration uses more efficient background
```

Comment:
This seems to just be a bald-faced lie. We totally ignore the border radius when drawing the background image, as far as I can tell.

Maybe that's another bug? We don't clip the background image to the border radius?